kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml

```diff
@@ -0,0 +1,15 @@
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: back-end
+spec:
+  template:
+    spec:
+      containers:
+      - name: app
+        env:
+        - name: DB_HOST
+          value: # TODO
+        - name: ENVIRONMENT
+          value: aws_prod
```

Comment:
what's the significance of this `aws_prod` vs `prod`?

Reply:
It's a hacky way to tell django to use the aws_s3 bucket to serve static assets, while still allowing me to run the production config locally sans remote static files